CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 ### Additions and Improvements
 - Add discovery options to genesis file [#2944](https://github.com/hyperledger/besu/pull/2944)
+- Add support for additional JWT authentication algorithms [#3017](https://github.com/hyperledger/besu/pull/3017) 
```

Comment:
you'll need to move this to 21.10.3 now :)

Reply:
wonder if this should be an experimental CLI option?

Reply:
2db5447